eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -214,15 +216,17 @@ function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation, $packageInfo)
   Set-Content -Path $readmeLocation -Value $outputReadmeContent
 }
 
+# For daily update and release, validate DocsMS publishing using the language-specific validation function
+if ($ValidateDocsMsPackagesFn -and (Test-Path "Function:$ValidateDocsMsPackagesFn")) {
+  Write-Host "Validating the packages..."
+
+  $packageInfos = @($PackageInfoJsonLocations | GetPackageInfoJson)
+
+  &$ValidateDocsMsPackagesFn -PackageInfos $packageInfos -PackageSourceOverride $PackageSourceOverride -DocValidationImageId $DocValidationImageId -DocRepoLocation $DocRepoLocation
+}
+
 foreach ($packageInfoLocation in $PackageInfoJsonLocations) {
   Write-Host "Updating metadata for package: $packageInfoLocation"
   # Convert package metadata json file to metadata json property.
-  $packageInfo = GetPackageInfoJson $packageInfoLocation
```

Comment:
We cannot remove this without breaking the other languages until they each are updated to start passing in this new parameter. What is our transition plan?

Reply:
Each of the language-settings files that implemented $ValidateDocsMsPackagesFn have been shimmed to support single packageInfo and multiple packageInfos parameters:
https://github.com/Azure/azure-sdk-for-js/pull/21483
https://github.com/Azure/azure-sdk-for-java/pull/28346
https://github.com/Azure/azure-sdk-for-python/pull/24041

Reply:
The yaml and powershell script for Update-DocsMsMetadata already supported multiple package info locations.  The implementation steps are:

1. Add batch support to the language repos (language repo prs)
2. Add batch invocation to eng/common (this pr)
3. Optimize batch processing per language
  - Pass multiple packageInfoLocations to [`update-docsms-metadata.yml`](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/pipelines/templates/steps/update-docsms-metadata.yml#L2)
  - Use deeper batch processing in `Validate-{language}-DocMsPackages`